build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.1'
+        classpath 'com.android.tools.build:gradle:4.1.0'
```

Comment:
should we call out this bump in dependencies in the CHANGELOG?

Reply:
Yeah Sammy had a similar question, maybe it makes sense to note dependency updates in the CHANGELOG going forward. It could bring dependencies more to the forefront of our process.

Reply:
Does upgrading the gradle version have any impact on the merchant integrating with Drop-in?

Reply:
It shouldn't. The Android Gradle Plugin is basically gradle's link into Android studio. Most merchants are compiling with the AAR library hosted by Nexus Sonatype.